code/modules/antagonists/heretic/knowledge/starting_lore.dm

```diff
@@ -293,3 +293,28 @@ GLOBAL_LIST_INIT(heretic_start_knowledge, initialize_starting_knowledge())
 	body.do_jitter_animation()
 	body.visible_message(span_danger("An awful ripping sound is heard as [ripped_thing]'s [exterior_text] is ripped straight out, wrapping around [le_book || "the book"], turning into an eldritch shade of blue!"))
 	return ..()
+
+/datum/heretic_knowledge/feast_of_owls
+	name = "Feast of Owls"
+	desc = "Allows you to undergo a ritual that gives you 5 knowledge points but locks you out of ascension. This can only be done once and cannot be reverted."
+	gain_text = "Under the soft glow of unreason there is a beast that stalks the night. I shall bring it forth and let it enter my presence. It will feast upon my amibitions and leave knowledge in it's wake."
```

Comment:
```suggestion
	gain_text = "Under the soft glow of unreason there is a beast that stalks the night. I shall bring it forth and let it enter my presence. It will feast upon my amibitions and leave knowledge in its wake."
```
'in it is wake'

Reply:
```suggestion
	gain_text = "Under the soft glow of unreason there is a beast that stalks the night. I brought it forth and let it enter my presence. It feasted upon my ambitions and left knowledge in its wake."
```
heretic gain text is usually in the context of a story from the past. if you want to make it fit it should be in past tense right?